editor/plugins/tile_set_editor_plugin.cpp

```diff
@@ -244,7 +244,7 @@ TileSetEditor::TileSetEditor(EditorNode *p_editor) {
 	MenuButton *options = memnew(MenuButton);
 	panel->add_child(options);
 	options->set_position(Point2(1, 1));
```

Comment:
What's the point of this? It makes it offset and unaligned.

Reply:
"Theme" button used to be at the bottom of the viewport, in an unreachable place, i dont know why, it was almost imposible to click, i moved it to the toolbar when was working on autotiles, this code is most probably from before o moved it, so just remove that line imo.